LICENSE

```diff
@@ -238,6 +241,9 @@ http://netty.io
 developed by The Netty Project
 License: Apache License v2 (http://www.apache.org/licenses)
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
```

Comment:
@mxm

Reply:
Yes, we use Netty heavily on the transport layer (via ES) and for our Postgres Wire Protocol.

Reply:
@mxm by "yes" do you mean yes it's a build time inclusion?

Reply:
If we use it heavily, it has to be a build time inclusion :)

Reply:
Just to clarify: We don't use the source of Netty, just the binary library.

Reply:
not necessarily! it could be bundled with the source. which is the only thing I need to determine for this work! :) but I have my answer now :)